api/odl.py

```diff
@@ -531,13 +571,21 @@ def update_hold_queue(self, licensepool):
         ).all()
 
         if len(holds) > remaining_licenses:
-            licensepool.licenses_available = 0
-            licensepool.licenses_reserved = remaining_licenses
-            licensepool.patrons_in_hold_queue = len(holds)
+            new_licenses_available = 0
+            new_licenses_reserved = remaining_licenses
+            new_patrons_in_hold_queue = len(holds)
         else:
-            licensepool.licenses_available = remaining_licenses - len(holds)
-            licensepool.licenses_reserved = len(holds)
-            licensepool.patrons_in_hold_queue = len(holds)
+            new_licenses_available = remaining_licenses - len(holds)
+            new_licenses_reserved = len(holds)
+            new_patrons_in_hold_queue = len(holds)
+        licensepool.update_availability(
+            licensepool.licenses_owned,
+            new_licenses_available,
+            new_licenses_reserved,
+            new_patrons_in_hold_queue,
+            analytics=self.analytics,
+            as_of=datetime.datetime.utcnow(),
+        )
```

Comment:
I thought I would have to add a `last_availability_update_time` to the crawlable feed order by but it turns out `LicensePool.update_availability` already changes `Work.last_update_time`.

Reply:
That does seem a little weird in isolation, but okay.